Source/SpeechToTextV1/SpeechToTextState.swift

```diff
@@ -17,6 +17,7 @@
 import Foundation
 
 /** The state of a `SpeechToTextSession`. */
+// swiftlint:disable identifier_name
```

Comment:
We should really change these to be lowercase, since that's the convention with Swift 3. Might actually be helpful to remove this exception so it appears as a warning in Xcode.

Reply:
Now that Xcode has refactor support for Swift, this seems to be straightforward.  I'll make this change.